### DISCUSSION TITLE
Need help for wol command

### DISCUSSION THREAD

**Discussion by davidande** [seriousm4x/UpSnap#525] [Help wanted]
hey,
trying to configure the wake up command from the gui.
cannot find a correct command to launch.
any help?
thanks :-)
David

**Comment by seriousm4x** [comment]
You only need that if you are not using the default WOL magic packet. Just leave it blank unless you have a specific reason why you can't use wol and have to resort to other methods.

Whatever you enter in the input field will get executed as a shell command.
